src/printer.js

```diff
@@ -49,6 +49,25 @@ function shouldPrintComma(options, level) {
   }
 }
 
+function getAlignmentSize(value, tabWidth, startIndex) {
```

Comment:
Sure, fixed in https://github.com/prettier/prettier/pull/1609/commits/db0ec7a6356a6c5d628b8c8c6542c61c67d9078c